apps/dashboard/src/components/workflow-editor/steps/in-app/in-app-editor.tsx

```diff
@@ -12,8 +12,8 @@ const redirectKey = 'redirect';
 const primaryActionKey = 'primaryAction';
 const secondaryActionKey = 'secondaryAction';
 
-export const InAppEditor = ({ uiSchema }: { uiSchema?: UiSchema }) => {
-  if (!uiSchema || uiSchema?.group !== UiSchemaGroupEnum.IN_APP) {
```

Comment:
Not sure if its better to check for `uiSchema` in child or parent - moved to parent for consistency with email-editor